tools/run_automation.sh

```diff
@@ -0,0 +1,14 @@
+#Create the virtual environment On macOS/Linux
+python3 -m venv myenv
+
+#Activate the virtual environment:
+source myenv/bin/activate
+
+# Install packages
+pip install -r requirements.txt
+
+# Enter the parameters: FILE_PATH_MENTORS_XLSX FILE_PATH_MENTORS_YML MODE
+# Example: samples/mentors.xlsx samples/mentors.yml a
+# mode "a" for APPEND new mentors from the xlsx table to the existing mentors.yml
+# mode "w" for WRITE all mentors from the xlsx table to mentors.yml
+python3 automation.py samples/mentors.xlsx samples/mentors.yml a
```

Comment:
python3 automation.py samples/mentors.xlsx samples/mentors.yml a **0**

I would also add a default value for skip_rows field. The problem is that the 'w' mode is never reached in case the skip_rows is missing (since there is a check for the number or args passed and if less than 5 then it defaults to the append mode). Or maybe, to implement some more logic in the code and to leave the setting as it is. We can decide. 

Reply:
```suggestion
python3 automation.py samples/mentors.xlsx samples/mentors.yml a 1
```